setup.py

```diff
@@ -7,7 +7,7 @@
 import subprocess
 
 from distutils.spawn import find_executable
-from distutils.core import DistutilsOptionError
+from distutils.errors import DistutilsOptionError
```

Comment:
distutils is deprecated package and will be removed from stdlib of future Python versions. We need to replace it with setuptools.